content/en/docs/started/installing-kubeflow.md

```diff
@@ -5,30 +5,160 @@ weight = 20
 
 +++
 
-## What is Kubeflow?
+This page describes how to install Kubeflow standalone components or Kubeflow Platform using package
+distributions or raw manifests.
```

Comment:
this might be rewritten or removed

Reply:
@jbottum Any suggestion how do you think we should re-write it ? I just tried to be consistent with other installation page that we updated with @StefanoFioravanzo and @hbelmiro. E.g. [Training Operator](https://www.kubeflow.org/docs/components/training/installation/) or [Katib](https://www.kubeflow.org/docs/components/katib/installation/).